src/mapml/layers/MapMLLayer.js

```diff
@@ -499,17 +499,21 @@ export var MapMLLayer = L.Layer.extend({
         extent.setAttribute('aria-labelledby', extentItemNameSpan.id);
         extentItemNameSpan.extent = this._extent._mapExtents[i];
 
-        extent.onmousedown = (downEvent) => {
-          if(downEvent.target.tagName.toLowerCase() === "input" || downEvent.target.tagName.toLowerCase() === "select") return;
-          downEvent.preventDefault();
+        extent.ontouchstart = extent.onmousedown = (downEvent) => {
+          if((downEvent.target.parentElement.tagName.toLowerCase() === 'label' &&
+              downEvent.target.tagName.toLowerCase() !== 'input') ||
+              downEvent.target.tagName.toLowerCase() === 'label') {          
+          // downEvent.preventDefault();
           downEvent.stopPropagation();
+          downEvent = downEvent instanceof TouchEvent? downEvent.touches[0] : downEvent;
 
           let control = extent,
               controls = extent.parentNode,
               moving = false, yPos = downEvent.clientY;
 
-              document.body.onmousemove = (moveEvent) => {
+              document.body.ontouchmove = document.body.onmousemove = (moveEvent) => {
```

Comment:
I didn't realize you could do assignments like this. Interesting and concise.

Reply:
I believe it is called chaining assignment (there is an example of it in mdn: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Assignment). it really helpful in saving time and space!

Reply:
New knowledge thanks!